doc/uftrace-replay.md

```diff
@@ -177,6 +177,31 @@ You can also see replay output with different time threshold for the same record
        6.448 us [ 1234] |   a();
        8.631 us [ 1234] | } /* main */
 
+In addition, The `-r` option can show functions executed within the given time range.
+When using this option by default, you can see TIMESTAMP or ELAPSED fields as well as DURATION and TID.
```

Comment:
I think it'd be better removing the "by default" in the second sentence, thanks!

Reply:
Yes. I amended!